R/openmp.R

```diff
@@ -0,0 +1,22 @@
+## We should have the system report back if it supports openmp at all,
+## and pass that in here too, because that deserves a warning.
```

Comment:
Are there systems (ie, toolchains) that don't support openMP now? Wonder how we detect this...

Reply:
yeah, the mac toolchain is still a mess